build.gradle

```diff
@@ -2,11 +2,15 @@
 buildscript {
   repositories {
     jcenter()
+    maven {
+      url 'http://oss.jfrog.org/artifactory/plugins-release'
+    }
```

Comment:
Is it possible to add the URL only when publishing the snapshots? Otherwise, Gradle is going to attempt to resolve jars in the snapshot repository whenever it runs.